### PR TITLE
fix some crowbar-from-git parts

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1198,20 +1204,17 @@ EOF
 
 function onadmin_installcrowbarfromgit()
 {
-    do_installcrowbar \
-        "CROWBAR_FROM_GIT=1
-            /opt/dell/bin/install-chef-suse.sh --from-git --verbose"
+    if iscloudver 5plus ; then
```
Comment (bkutil) [inline scripts/qa_crowbarsetup.sh]: This probably deserves a comment. Why is this not handled by update-alternatives/rpm script?

**Comment by bkutil** [comment]
+1

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: I guess, that is a long story, that nobody told me yet. kkaempf did some controversial things there.

Comment (tboerger) [inline scripts/qa_crowbarsetup.sh]: On SLE 11 ruby does not have update-alternatives.

**Comment by jdsn** [comment]
:+1:

**Pull request merged**